### PR TITLE
update NYSOASAS Agency name, update the subgroup to reflect source desired change, and update operator name to provider name

### LINKED ISSUE #597: Changes to NYS OASAS facilities
- [x] Update the agency name "NYS Office of Alcoholism and Substance Abuse Services" to "NYS Office of Addiction Services and Supports"
- [x] Update the subgroup value "Chemical Dependency" to "Substance Use Disorder Treatment Programs"
- [x] Change the "Operated by" value to the Provider name of the program

Data provider linked to [here](https://webapps.oasas.ny.gov/providerDirectory/index.cfm) to retrieve data and recommended updating data weekly.

### PR DESCRIPTION
**Pull request by mbh329**
Addresses issue #597.

This PR addresses 3 issues identified by a source at NYSOASAS who reached out to us to change:
* Agency Name to NYS Office of Addiction Services and Supports 
* Clarify that OASAS does not operate these facilities therefore they would like the operator name in the app to reflect the provider name given in the source data. 
* Change the wording on the Facilities DB subgroup category to reflect updated terminology 

Note:
* Based on the screenshot they provided us with (see below). Based on the structure of the data, I think the updated `Overseen By` issue they flagged in the email would have to be changed by OSE but the `Operated By` issue should be a like for like replacement based on the current data schema and structure.
image001

Testing:
* open up the facilities repo and follow the the steps in the README.md to build 

Let me know if there are any questions!

### CODE REVIEW

File: facdb/sql/nysoasas_programs.sql [hunk 1]
```diff
@@ -14,8 +14,8 @@ SELECT
     NULL as bin,
     NULL as bbl,
     service_type as factype,
-    'Chemical Dependency' as facsubgrp,
-    'NYS Office of Alcoholism and Substance Abuse Services' as opname,
+    'Substance Use Disorder Treatment Programs' as facsubgrp,
+    provider_name as opname,
```
Comment (AmandaDoyle) [inline facdb/sql/nysoasas_programs.sql:18]: Do we need to implement an `initcap` function here?  That's the only question I have / thing that stands out.  Everything else looks good.

**Review by damonmcc — APPROVED** [review]
lgtm!

seems risky that the value has to be changed in more than one place. but that's a design enhancement thought, definitely not worth doing here

File: facdb/sql/nysoasas_programs.sql (hunk 1 above)
Comment (mbh329) [reply facdb/sql/nysoasas_programs.sql:18]: the initcap function would set the acronym's to lowercase. As it stands, the source data seems to be in the proper format (the below output is an example of records without the initcap function):
Screenshot 2023-02-06 at 1 12 20 PM

Comment (mbh329) [reply facdb/sql/nysoasas_programs.sql:18]: With the initcap it would look like this:
Screenshot 2023-02-06 at 1 14 54 PM

**Review by AmandaDoyle — APPROVED** [review]

**Pull request merged**